packages/backend/src/apps/lettersg/helpers/attachment.ts

```diff
@@ -0,0 +1,27 @@
+import { IGlobalVariable } from '@plumber/types'
```

Comment:
nit: i like to import types if the underlying impl is not needed
```suggestion
import { type IGlobalVariable } from '@plumber/types'
```

Reply:
sian do you change this manually or is this auto imported